src/test/java/seedu/address/logic/parser/SuggestCommandParserTest.java

```diff
@@ -19,18 +18,17 @@
 
     @Test
     public void parse_validArgs_returnsSuggestCommand() {
-        Predicate<Client> availablePredicate = new SuggestionType("available").getSuggestionPredicate();
-        assertParseSuccess(parser, "suggest by/available",
-                new SuggestCommand(Collections.singletonList(availablePredicate)));
+        Set<SuggestionType> suggestionTypes = new LinkedHashSet<>();
+        suggestionTypes.add(new SuggestionType(SuggestionType.BY_AVAILABLE));
+        assertParseSuccess(parser, "suggest by/available", new SuggestCommand(suggestionTypes));
```

Comment:
In the actual case, the command word "suggest" shouldn't be passed into the parser. The `SuggestCommandParser` should have an additional check for non-empty preambles which was not added when I did created the skeleton initially. E.g. `suggest 1243 adsfkn by/available` is still valid now but it probably shouldn't be.

Reply:
done